admin_guide/allocating_node_resources.adoc

```diff
@@ -101,6 +101,50 @@ status:
 ----
 ====
 
+[[system-resources-reported-by-kubelet]]
+== System resources reported by kubelet
+
+Since openshift 3.3 the kubelet reports system resources utilized by the container runtime and kubelet.
+To better aid your ability to configure *--system-reserved* and *--kube-reserved*,
+you can introspect the nodes resource usage using the kubelet summary api which is accessible at *<node>:<port>/stats/summary*.
+
+To access the API from localhost with kubelet listening on default port, you can run:
+
+----
+$ curl -k https://127.0.0.1:10250/stats/summary
```

Comment:
I just tested origin master locally, and I see kubelet and runtime, when not launched from a unit.